app/src/main/java/com/google/android/soundchecker/AudioEncoderDecoderActivity.kt

```diff
@@ -1221,6 +1311,7 @@ class AudioEncoderDecoderActivity : ComponentActivity() {
         private val AUDIO_FORMAT_AAC = MediaFormat.MIMETYPE_AUDIO_AAC
         private val DEFAULT_SAMPLE_RATES = listOf(8000, 16000, 32000, 44100, 48000, 96000, 192000)
         private val DEFAULT_BITRATES = listOf(6000, 10000, 20000, 64000, 128000)
+        private val ENCODER_DELAYS = listOf(0, 1000, 2000, 4000, 8000, 16000)
```

Comment:
Could you also document how the delays value are defined?

Reply:
Done! Thanks!